tests/request_replayer/index.php

```diff
@@ -1,15 +1,21 @@
 <?php
 
-$requestLog = '/tmp/php_request_replayer_' . getmypid() . '.json';
+$defaultRequestLog = '/tmp/php_request_replayer_' . getmypid() . '.json';
+$requestLog = getenv('DD_REQUEST_DUMPER_FILE') ? : $defaultRequestLog;
```

Comment:
And then we can change this such that `DD_REQUEST_DUMPER_FILE` is just the file name jailed to the system temp dir. :)
```suggestion
if (getenv('DD_REQUEST_DUMPER_FILE')) {
    $requestLog = sys_get_temp_dir() . '/' . basename(getenv('DD_REQUEST_DUMPER_FILE'));
}
```